### ISSUE TITLE
fails to install in rebar3

### ISSUE THREAD

**Issue by ityonemo** [michalmuskala/jason#163]
adding the dependency in a rebar3 project using:

```
{plugins, [rebar_mix]}.
{provider_hooks, [
  {post, [{compile, {mix, consolidate_protocols}}]}
]}.
{deps, [jason]}.
```

fails due to the error:  `Dep decimal has invalid version ~> 1.0 or ~> 2.0`

is this a Jason problem or is this a rebar3 problem?

**Comment by michalmuskala** [comment]
This looks like a rebar3 problem. `~> 1.0 or ~> 2.0` is a valid version constraint in mix

**Comment by filmor** [comment]
This is "fixed" in rebar3 3.23, https://github.com/erlang/rebar3/pull/2840.
